src/cdomains/intDomain.ml

```diff
@@ -1314,9 +1307,9 @@ struct
     (* A definite value is leq all exclusion sets that don't contain it *)
     | `Definite x, `Excluded (s,r) -> in_range r x && not (S.mem x s)
     (* No finite exclusion set can be leq than a definite value *)
-    | `Excluded _, `Definite _ -> false
+    | `Excluded _, `Definite _ -> false (* wrong *)
     (* Excluding X <= Excluding Y whenever Y <= X *)
-    | `Excluded (x,xw), `Excluded (y,yw) -> S.subset y x && R.leq xw yw
+    | `Excluded (x,xw), `Excluded (y,yw) -> S.subset y x && R.leq xw yw (* wrong *)
```

Comment:
I'm guessing it's a draft because of these "wrongs"?

Reply:
Yes, good catch!